src/Client/Webhook.php

```diff
@@ -118,11 +118,17 @@ public function redeliverDelivery(string $storeId, string $webhookId, string $de
         }
     }
 
-    public function createWebhook(string $storeId, string $url, ?array $specificEvents, ?string $secret): \BTCPayServer\Result\WebhookCreated
-    {
+    public function createWebhook(
+        string $storeId,
+        string $url,
+        ?array $specificEvents,
+        ?string $secret,
+        ?bool $enabled = true,
```

Comment:
Remove the '?'
What would happen if you supply 'null' as argument?

Reply:
Does this work? I thought optional params need that but thinking about it you might be right, will check and test.

Reply:
fixed with commit in March 😬